src/Serializer/AbstractItemNormalizer.php

```diff
@@ -308,7 +309,17 @@ protected function denormalizeRelation(string $attributeName, PropertyMetadata $
             $context['resource_class'] = $className;
             $context['api_allow_update'] = true;
 
-            return $this->serializer->denormalize($value, $className, $format, $context);
+            try {
+                return $this->serializer->denormalize($value, $className, $format, $context);
+            } catch (InvalidValueException $e) {
+                if ('invalid IRI?' !== $e->getReason()) {
```

Comment:
Well, wasn't sure if we should have an invalid value exception for the invalid iri, but I could alter that to use one instead ?

Reply:
Not sure I got that. This exception is only thrown when IRI is invalid right?

Reply:
Yeah, but the part between parentheses looked variable, that's why I added a reason. I can merge it if needed

Reply:
Well, we had `Invalid value (Invalid IRI?).`, so I thought that we could have multiple `Invalid Value (reason)`, and it could sometimes be `Invalid Value (Invalid IRI?)`. That's all, but then, I can make a `InvalidIriException` and be done with it :P

Reply:
Yes please :D